Web/starter_project_barbies/components/about/HeroSection.tsx

```diff
@@ -0,0 +1,63 @@
+import Image from 'next/image'
+
+const HeroSection = ({}) => {
+  return (
+    <div className="flex justify-center gap-10 flex-wrap">
+      <div className="flex flex-col justify-center">
+        <div className="font-bold md:text-2xl sm:text-xl lg:text-3xl xl:text-4xl p-3">
+          <span className="text-blue-500">Africa </span>to Silicon Valley
```

Comment:
Let's define colors globally(tailwind.config.js) so others can also use it.

Reply:
i think you should be using the primary color here as well (make sure the primary is defined appropriately too)
because the current color feels a bit off in the UI